plugins/modules/ovirt_disk.py

```diff
@@ -366,27 +372,27 @@ def transfer(connection, module, direction, transfer_func):
             time.sleep(module.params['poll_interval'])
             transfer = transfer_service.get()
 
-        proxy_url = urlparse(transfer.proxy_url)
-        context = ssl.create_default_context()
         auth = module.params['auth']
-        if auth.get('insecure'):
-            context.check_hostname = False
-            context.verify_mode = ssl.CERT_NONE
-        elif auth.get('ca_file'):
-            context.load_verify_locations(cafile=auth.get('ca_file'))
-
-        proxy_connection = HTTPSConnection(
-            proxy_url.hostname,
-            proxy_url.port,
-            context=context,
-        )
-
-        transfer_func(
-            transfer_service,
-            proxy_connection,
-            proxy_url,
-            transfer.signed_ticket
-        )
+        if module.params['use_proxy']:
+            destination_url = transfer.proxy_url
+        else:
+            destination_url = transfer.transfer_url
+        if direction == otypes.ImageTransferDirection.DOWNLOAD:
+            client.download(
+                destination_url,
+                module.params['download_image_path'],
+                auth.get('ca_file'),
+                fmt='qcow2' if module.params['format'] == 'cow' else 'raw',
+                incremental=module.params['format'] == 'cow',
```

Comment:
incremental is relevant only to incremental backup, which is not related to the format argument. Please remove this.